content/zh-cn/blog/_posts/2022-04-07-Kubernetes-1-24-removals-and-deprecations.md

```diff
@@ -91,10 +91,10 @@ and the [updated dockershim removal FAQ](/blog/2022/02/17/dockershim-faq/).
 Take a look at the [Is Your Cluster Ready for v1.24?](/blog/2022/03/31/ready-for-dockershim-removal/) post to learn about how to ensure your cluster continues to work after upgrading from v1.23 to v1.24. 
 -->
 有关 Kubernetes 为何不再使用 dockershim 的更多信息，
-请参见：[Kubernetes 正在离开 Dockershim](/blog/2022/01/07/kubernetes-is-moving-on-from-dockershim/)
+请参见：[Kubernetes 正在离开 Dockershim](/zh-cn/blog/2022/01/07/kubernetes-is-moving-on-from-dockershim/)
 和[最新的弃用 Dockershim 的常见问题](/zh-cn/blog/2022/02/17/dockershim-faq/)。
 
-查看[你的集群准备好使用 v1.24 了吗？](/blog/2022/03/31/ready-for-dockershim-removal/) 一文，
+查看[你的集群准备好使用 v1.24 了吗？](/zh-cn/blog/2022/03/31/ready-for-dockershim-removal/) 一文，
```

Comment:
Use the same title as [that article](https://kubernetes.io/zh-cn/blog/2022/03/31/ready-for-dockershim-removal/).
```suggestion
查看[你的集群准备好使用 v1.24 版本了吗？](/zh-cn/blog/2022/03/31/ready-for-dockershim-removal/) 一文，
```